### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by toshiakitakase**

### CODE REVIEW

File: frontend/packages/react-app/src/components/molecules/TopBody.jsx [hunk 1]
```diff
@@ -4,24 +4,28 @@ import { Button, Heading, Card, Text, Flex } from "rimble-ui";
...
-      <Card mt={3}>
-        <Heading mt={0} fontSize={5} color="itblue">For fans</Heading>
-        <Text mb={3}>
-          Check out the tokens of your favorite creators and see if you can connect with them.
-        </Text>
-        <Link style={{textDecoration: 'none'}} to="/explore"><Button mainColor="#48C5D5">Explore</Button></Link>
-      </Card>
-    <Heading as={"h3"} mt={4}>Get token from your Audius account</Heading>
-    <Link to="/audius-token"><Button>get token with Audius</Button></Link>
+    <Heading fontSize={6} bg={"white"}>Fans and creators can connect through social moneys.</Heading>
```
Comment (wild-mouse) [inline frontend/packages/react-app/src/components/molecules/TopBody.jsx:7]: I prefer no background color for seamless background image. How do you thinki?

**Comment by wild-mouse** [comment]
Creators audius dashboard page seems not styled yet. Will you update it in other PRs?

**Comment by toshiakitakase** [comment]
Mistake! I will do it now.

**Comment by toshiakitakase** [comment]
DONE!

**Review by wild-mouse — APPROVED** [review]
LGTM 🏃‍♂️

**Pull request merged**